factories/esSearcherFactory.js

```diff
@@ -77,7 +77,7 @@
       }
 
       var remaining       = self.numFound - pagerArgs.from;
-      pagerArgs.size      = Math.min(10, remaining);
+      pagerArgs.size      = Math.min(pagerArgs.size, remaining);
```

Comment:
not quite. it does specify the `size` param because the default is 10, but for snapshots we're sending an array of ids that's bigger than 10, so we need to be able to set the `size: ids.length`.